src/index.ts

```diff
@@ -673,6 +682,98 @@ export class ERC725 {
   encodeKeyName(keyName: string, dynamicKeyParts?: DynamicKeyParts): string {
     return encodeKeyName(keyName, dynamicKeyParts);
   }
-}
 
+  baseConfig = { ipfsGateway: IPFS_GATEWAY_BASE_URL };
+
+  lspTypeOptions: Record<Exclude<LSPType, LSPType.Unknown>, LspTypeOption> = {
```

Comment:
What is this object? It just contains the interface id which we already have in a constant and the SupportedStandard ABI which should be the same for each standard anyway? I think we can simplify this and do without this object

Reply:
We wanted to have one object that can hold interfaceId but also Schema, so the function works pretty generic, and we just check if its a schema or interface standard- and then we basically map everything thats needed onto the LSPs.